### PR TITLE
Feature/game design

### PR DESCRIPTION
**Pull request by baptiste-arnoulet**
Principal structure of the race and finish Game Design

### CODE REVIEW

File: Projet/render/Shader.h [hunk 1]
```diff
@@ -47,7 +47,10 @@ struct Shader {
 	ID3D11Buffer* PPointLightsBuffer;
 	ID3D11ShaderResourceView* PPointLightsBufferView;
 
+	//Shader(const wchar_t* FileName);
+
```
Comment (baptistegasser) [inline Projet/render/Shader.h]: bruh

Comment (baptistegasser) [reply Projet/render/Shader.h]: remove commented code *

Comment (baptiste-arnoulet) [reply Projet/render/Shader.h]: good

**Review by carobouette — APPROVED** [review]
paf la girafe

**Pull request merged**